src/esbmc/bmc.cpp

```diff
@@ -853,5 +859,109 @@ smt_convt::resultt bmct::multi_property_check(
       log_result("Assertion Instances Coverage: 0%");
   }
 
+  // Condition Coverage:
+  else if (is_cond_cov)
+  {
+    log_success("\n[Coverage]\n");
+
+    goto_coveraget tmp(ns, symex->goto_functions);
+    const std::unordered_set<std::string> &total_cond_assert =
+      tmp.get_total_cond_assert();
+    int tracked_instance = 0;
+    int total_instance = 0;
+    std::unordered_set<std::string> short_circuit_instance = {};
+    std::unordered_set<std::string> reached_instance_loc = {};
+
+    // show claims
+    bool cond_show_claims =
+      options.get_bool_option("condition-coverage-claims");
+
+    // reached claims:
+    auto total_cond_assert_cpy = total_cond_assert;
+    for (const auto &claim : total_cond_assert)
+    {
+      if (reached_claims.count(claim))
+      {
+        // update counter
+        total_instance += 2;
+
+        if (cond_show_claims)
+          log_status("  {} : SATISFIED", claim);
+        tracked_instance++;
+        reached_claims.erase(claim);
+        total_cond_assert_cpy.erase(claim);
+
+        // reversal
+        std::string delimiter = "\t";
+        auto pos = claim.find(delimiter);
+        std::string claim_msg = claim.substr(0, pos);
+        std::string claim_loc = claim.substr(pos + 1);
```

Comment:
This code deconstructs the conceptual "pair" from the above remark and makes the assumption that `"\t"` is unique. Both would not be needed if it was a pair (or the iterator into the goto-instructions) in the first place.